test/libsolidity/SyntaxTest.cpp

```diff
@@ -69,7 +75,11 @@ TestCase::TestResult SyntaxTest::run(ostream& _stream, string const& _linePrefix
 	compiler().reset();
 	compiler().setSources({{"", versionPragma + m_source}});
 	compiler().setEVMVersion(m_evmVersion);
-
+	compiler().setOptimiserSettings(
+		m_optimiseYul ?
+		OptimiserSettings::full() :
```

Comment:
Since this setting was not there before, does it mean all yul syntax tests are now run with full optimization turned on?

Reply:
It is a setting for the solidity syntax tests. If the setting is activated, full optimization (including yul) is set, otherwise no optimization. Note that syntax tests do not really run the optimizations, it is just the setting that will trigger the error here: https://github.com/ethereum/solidity/pull/6840/files#diff-fdb8e768955854a054159ad58777aa86R12

Reply:
Ah I see. Thanks for the clarification!